### DISCUSSION TITLE
Data Importer in Pimcore X - setOmitMandatoryCheck

### DISCUSSION THREAD

**Discussion by judip** [pimcore/pimcore#10335] [Ideas]
It is necessary to add an option "setOmitMandatoryCheck" to configuration, because sometimes import data without all fields filled. Currently it shows Pimcore\Model\Element\ValidationException: Empty mandatory field and object is not created. This is a big problem.

**Comment by fashxp** [comment]
for now you could use Events to set that programatically, see https://pimcore.com/docs/data-importer/1.3/Extending/Events.html 

but I agree, could be a useful setting for import configuration. maybe add a mandatoryCheck strategy similar to the publish strategy, with strategies like `alwaysCheck`, `neverCheck`, `checkWhenPublished` ... 

we would love to see a PR for that.
